File: functions/check_org.js
const { initializeApp, cert } = require('firebase-admin/app');
const { getFirestore } = require('firebase-admin/firestore');
const serviceAccount = require('../sa.json');

initializeApp({ credential: cert(serviceAccount) });
const db = getFirestore();

async function checkOrg() {
  const doc = await db.collection('organizations').doc('default').get();
  if (doc.exists) {
    console.log('Default Org Data:', JSON.stringify(doc.data(), null, 2));
  } else {
    console.log('Default org does not exist!');
  }
}

checkOrg();

File: check_org.py
import firebase_admin
from firebase_admin import credentials
from firebase_admin import firestore
import json

cred = credentials.Certificate('sa.json')
firebase_admin.initialize_app(cred)

db = firestore.client()
doc = db.collection('organizations').document('default').get()

if doc.exists:
    print(json.dumps(doc.to_dict(), indent=2))
else:
    print('Default org does not exist!')
